Assets/Scripts/Drone Scripts/Drone.cs

```diff
@@ -260,33 +288,33 @@ public int WaypointsCount()
         }
 
         /// <summary>
-        /// Return the number of deleted waypoints
+        /// The number of waypoints that were placed, but got deleted for the current drone.
```

Comment:
Does the script make a difference between current drone and non current drone?

Reply:
yep, each drone has it's own drone.cs script